app/src/test/java/org/simple/clinic/drugs/EditMedicineEffectHandlerTest.kt

```diff
@@ -144,4 +148,15 @@ class EditMedicineEffectHandlerTest {
 
     verifyZeroInteractions(uiActions)
   }
+
+  @Test
+  fun `when load drug frequency choice items effect is received, then load drug frequency choice items`() {
+    // when
+    testCase.dispatch(LoadDrugFrequencyChoiceItems)
+
+    // then
+    val expectedResult = drugFrequencyFactory.provideFields()
```

Comment:
We're not testing whether the `drugFrequencyFactory.provideFields()` is returning list of choice items or not. We're just adding the expected list and expecting it to work fine. Can you add this, please?